packages/server/config/database.js

```diff
@@ -1,5 +1,5 @@
 const mongoose = require("mongoose");
-require("dotenv").config();
+require("dotenv").config({path: './config/.env'});
```

Comment:
There shouldn't be a need to specify an env in the config file.
What was the bug you were getting? This should be solved by adding an env in packages/server